index.html

```diff
@@ -83,6 +83,204 @@ <h1>
         </ul>
       </div>
       </section>
+
+      <!-- Tonic -->
+
+      <section class="grid">
+        <div class="flex">
```

Comment:
- [ ] Kindly ensure there is a spacing between the `headline section` and the `projects section` to match the original design

![image](https://user-images.githubusercontent.com/98453979/219641215-e14f2312-fd57-4dca-b324-e37251ecd1bc.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219641455-c0dde6fe-5c68-4ed5-9e58-7ea38274bd17.png)